kolibri/core/assets/src/views/HorizontalNavBarWithOverflowMenu.vue

```diff
@@ -122,7 +125,17 @@
     display: flex;
     flex-direction: row;
     flex-wrap: wrap;
-    overflow: hidden;
```

Comment:
I think this change should be reverted - in my local testing, this didn't change any of the behaviour, and I think overflow is more generally applicable than text-overflow.